src/SortableContainer/index.js

```diff
@@ -124,7 +126,7 @@ export default function sortableContainer(
     handleStart = (event) => {
       const {distance, shouldCancelStart} = this.props;
 
-      if (event.button === 2 || shouldCancelStart(event)) {
+      if (event.button !== LEFT_MOUSE_BUTTON || shouldCancelStart(event)) {
```

Comment:
Please note: event.button is well supported: https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/button, and could be a number 0-4.  But, we should only activate for left mouse button.